raisinbread/RB_files/RB_conflicts_unresolved.sql

```diff
@@ -574,9 +574,6 @@ INSERT INTO `conflicts_unresolved` (`ConflictID`, `TableName`, `ExtraKeyColumn`,
 INSERT INTO `conflicts_unresolved` (`ConflictID`, `TableName`, `ExtraKeyColumn`, `ExtraKey1`, `ExtraKey2`, `FieldName`, `CommentId1`, `Value1`, `CommentId2`, `Value2`) VALUES (664,'radiology_review',NULL,' ',' ','Examiner','300192OTT192192111471371836','10','DDE_300192OTT192192111471371836','17');
 INSERT INTO `conflicts_unresolved` (`ConflictID`, `TableName`, `ExtraKeyColumn`, `ExtraKey1`, `ExtraKey2`, `FieldName`, `CommentId1`, `Value1`, `CommentId2`, `Value2`) VALUES (665,'radiology_review',NULL,' ',' ','Date_taken','300192OTT192192111471371836','2014-07-05','DDE_300192OTT192192111471371836','2014-06-03');
 INSERT INTO `conflicts_unresolved` (`ConflictID`, `TableName`, `ExtraKeyColumn`, `ExtraKey1`, `ExtraKey2`, `FieldName`, `CommentId1`, `Value1`, `CommentId2`, `Value2`) VALUES (666,'radiology_review',NULL,' ',' ','Review_date','300192OTT192192111471371836','2014-06-02','DDE_300192OTT192192111471371836','2014-06-01');
-INSERT INTO `conflicts_unresolved` (`ConflictID`, `TableName`, `ExtraKeyColumn`, `ExtraKey1`, `ExtraKey2`, `FieldName`, `CommentId1`, `Value1`, `CommentId2`, `Value2`) VALUES (667,'radiology_review',NULL,' ',' ','Examiner','300203OTT2031190111471371963','10','DDE_300203OTT2031190111471371963','17');
```

Comment:
@cmadjar  
since these three records got removed, 
in this file [modules/dashboard/test/DashboardTest.php     line 472 ]
577 change to 574
it will work.